code/modules/mob/living/inventory.dm

```diff
@@ -17,7 +17,7 @@
 	var/list/inventory_slots
 
 /mob/living/get_inventory_slots()
-	return inventory_slots
+	return global.all_inventory_slots // inventory_slots
```

Comment:
As discussed on Discord this won't work since the code that (used to) use `get_inventory_slots()` expected an associative list of slot name -> inventory datum. That said, not really a blocking issue since `get_inventory_slots()` is no longer used.